### ISSUE TITLE
Extend to other converters

### ISSUE THREAD

**Issue by keinstein** [mist13/ABC_ImpEx#1]
Hi, am I right that it is not so hard to extend this plugin to use other command line musicxml-to-anything and anything-to-musicxml converters like xml2ly?

What must be done for that?

**Comment by mist13** [comment]
You are right, the basis for such extensions should be there. A few things to consider:
How do you piece together the command line for your converter? (This plugin's code should offer enough guidance.)
Do you need to save stuff to temp files? (I needed to do that quite a bit) If you want to go via standard in/out, you may run into problems in different OS's.
Do the tools involved speak UTF-8 out of the box or do you need to do some trickery?
Does all of that work on all platforms supported by MuseScore?
Happy coding!
